sdk/data_cosmos/src/resources/document/indexing_directive.rs

```diff
@@ -13,6 +13,12 @@ pub enum IndexingDirective {
     Exclude,
 }
 
+impl Default for IndexingDirective {
```

Comment:
Consider using `derive_default_enum` with a bumped MSRV?  Given the soon-to-be-stabilized IntoFuture, we're likely to use this in the future.

Reply:
We can switch to that when we bump the MSRV for `IntoFuture`.